src/generic/sub-header/SubHeader.scss

```diff
@@ -1,6 +1,12 @@
 .sub-header {
   display: flex;
 
+  .sub-header-title-actions {
+    .pgn__menu-item:focus {
+      font-weight: bold;
+    }
+  }
+
```

Comment:
Without this, the focused text is too bold:

![Screen Shot 2023-12-13 at 6 18 05 PM](https://github.com/openedx/frontend-app-course-authoring/assets/6829768/9899fc5e-ba7b-47a7-90ac-d4bb3a1749a8)

Reply:
@yusuf-musleh I think this style is specific to the taxonomy listing's SelectMenu. It should not go in the generic SubHeader styles. They must go together with the specific component. And there put in a comment the reason that you already mentioned

Reply:
@ChrisChV Makes sense, I've created a file for `TaxonomyListPage` styles and moved it there: https://github.com/openedx/frontend-app-course-authoring/commit/c42e0892121655f4fa54a950fe2bef99614e5716